packages/material-ui/src/Popper/Popper.js

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { createPopper } from '@popperjs/core';
+import { useTheme } from '@material-ui/core/styles';
```

Comment:
Should we use a relative import to avoid bundlers being confused if there are two versions of the package available?
```suggestion
import { useTheme } from '../styles';
```